### PR TITLE
Feature/lattice 1269 entity type entity set collections

### PR DESCRIPTION
**Pull request by katherinebernstein**

### CODE REVIEW

**Review by you-leee — APPROVED** [review]
🙌 Cool refactor!

**Review by makosblade — APPROVED** [review]

**Review by geekbeast — APPROVED** [review]

File: src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java [hunk 1]
```diff
@@ -1618,8 +1503,7 @@ public boolean verifyElasticsearchConnection() {
         return connected;
     }
 
-    @Scheduled(
-            fixedRate = 1800000 )
+    @Scheduled( fixedRate = 1800000 )
     public void verifyRunner() throws UnknownHostException {
         verifyElasticsearchConnection();
```
Comment (geekbeast) [inline src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java:1500]: I've always wondered if this is the recommended way of doing this.

File: src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java [hunk 2]
```diff
@@ -1574,7 +1451,7 @@ private SearchResult executeFQNSearch(
     public boolean triggerIndex(
             String index,
             String type,
-            Iterable<? extends Object> objects,
+            Iterable<?> objects,
```
Comment (geekbeast) [inline src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java:1446]: eggcelent

File: src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java [hunk 3]
```diff
@@ -754,15 +690,14 @@ public boolean clearEntitySetData( UUID entitySetId, UUID entityTypeId ) {
...
-//    private EntityDataKeySearchResult getEntityDataKeySearchResult( SearchResponse response ) {
-//        Set<EntityDataKey> entityDataKeys = Sets.newHashSet();
-//        for ( SearchHit hit : response.getHits() ) {
-//            entityDataKeys.add( new EntityDataKey( getEntitySetIdFromHit( hit ),
-//                    UUID.fromString( hit.getId() ) ) );
-//        }
-//        return new EntityDataKeySearchResult( response.getHits().getTotalHits(), entityDataKeys );
-//    }
-
+    //    private EntityDataKeySearchResult getEntityDataKeySearchResult( SearchResponse response ) {
```
Comment (geekbeast) [inline src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java]: cleanup

**Review by geekbeast — CHANGES_REQUESTED** [review]

File: src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java (hunk 1 above)
Comment (katherinebernstein) [reply src/main/java/com/openlattice/kindling/search/ConductorElasticsearchImpl.java:1500]: same.... I *think* this originally came from one of elasticsearch's getting started guides, but that was a very long time ago

**Review by geekbeast — APPROVED** [review]

**Pull request merged**